phpcs.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0"?>
+<ruleset name="Seravo">
+  <description>
+    Seravo coding standards definition. Mostly WordPress coding standards, but relaxed a bit to be
+    easier on developers. This should be on-par with the Seravo Plugin phpcs.xml file.
+  </description>
+
+  <!-- Show sniff progress -->
+  <arg value="p"/>
+
+  <!-- All files should be UTF-8 encoded. -->
+  <arg name="encoding" value="utf-8"/>
+
+  <!-- check current and all subfolders if no file parameter given. -->
+  <file>.</file>
+
+  <!-- Don't check our installation script -->
+  <exclude-pattern>/wp-content/install.php</exclude-pattern>
+
+  <!-- Don't check composer dependencies and extras -->
+  <exclude-pattern>/vendor/</exclude-pattern>
+
+  <!-- Don't check WP core files -->
+  <exclude-pattern>/htdocs/wp-content/object-cache.php</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/object-cache.php.off</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/mu-plugins/bedrock-autoloader.php</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/mu-plugins/register-theme-directory.php</exclude-pattern>
+
+  <!--
+    Exclude all but plugins that are developed in this branch. PHPCS does not
+    support exclude+include so we need this list.
+  -->
+  <exclude-pattern>/htdocs/wp-content/mu-plugins/seravo-plugin</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/advanced-custom-fields</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/autodescription</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/google-analytics-dashboard-for-wp</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/index.php</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/mailpoet</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/maintenance</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/post-duplicator</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/redirection</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/simple-custom-post-order</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/simple-header-footer-html</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/wp-recaptcha-integration</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/caldera-forms</exclude-pattern>
+  <exclude-pattern>/htdocs/wp-content/plugins/native-lazyload</exclude-pattern>
```

Comment:
This list only applies to one site. Every site should have their own excludes. I believe that these should be replaced with a comment explaining that the user should exclude directories that they are not developing themselves.

Reply:
Updated.